Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -66,6 +66,15 @@ struct SettingsView: View {
                             Text(vehicle.model)
                         }
                     }
+                    .onDelete(perform: { indexSet in
+                        for index in indexSet {
+                            let vehicle = viewModel.vehicles[index]
+                            
+                            Task {
+                                await viewModel.deleteVehicle(vehicle)
+                            }
+                        }
+                    })
```

Comment:
Can you use instead specifically use this modifier `.swipeActions(edge: .trailing, allowsFullSwipe: true)`

So that it will be implemented the same way as the `Dashboard`

Reply:
Ok, let me change it

Reply:
I have changed onDelete to swipeActions. Kindly review it.